docs/source/user-guide/concepts/data-science.rst

```diff
@@ -0,0 +1,27 @@
+=========================
+Conda for data scientists
+=========================
+
+Conda is useful for any packaging process but it stands out from other
+package and environment management systems through its utility for data
+science.
+
+Conda’s benefits include:
+
+* Facilitating reproducibility of research workflows.
```

Comment:
Remove this, keep the similar line below about "allowing you to provide your environment..."